README.md

```diff
@@ -176,12 +181,3 @@ $ heroku drains:add https://logs.loggly.com/inputs/#{CUSTOMER_TOKEN}/tag/product
 This will tag all of the logs for the project with `production`, `elixir`, and
 the name of the project. This will allow you to more easily filter out the logs
 from other projects.
-
-## Dependencies
-
-- Erlang (latest)
-- Elixir (latest)
-- Hex
-- Rebar
-- Phoenix
-- Postgresql
```

Comment:
If we include dependencies in the README, we need to address these issues:
* The current list is a subset of a mix of the primary and transitive dependencies.
* Erlang is a dependency of Elixir.
* The Elixir dependency is implicit.
* The Erlang and Elixir versions are already specified in `.tool-versions`.
* Hex tasks are included with Mix, which is a part of Elixir.
* Rebar is packaged with Elixir.
* Phoenix is already in WebDelivery’s `mix.exs` deps.
* Postgresql is not a dependency.